### PR TITLE
media-sound/pms: migrate live ebuild to cmake (and update)

### PR DESCRIPTION
**Pull request by Hummer12007**
See ambientsound/pms#38, ambientsound/pms#41

### CODE REVIEW

File: media-sound/pms/pms-9999.ebuild [hunk 1]
```diff
@@ -39,17 +39,12 @@ pkg_pretend() {
 }
 
 src_prepare() {
-	eapply "${FILESDIR}/pms-9999-fix-automagic-dep.patch"
-	eapply_user
+	default
 
 	eautoreconf
```
Comment (a17r) [inline media-sound/pms/pms-9999.ebuild]: do you still need eautoreconf after removing the buildsystem patch or could you just drop the whole src_prepare?

Comment (Hummer12007) [inline media-sound/pms/pms-9999.ebuild]: Yes, it's required to generate the configure script (it's the live package).

Comment (a17r) [inline media-sound/pms/pms-9999.ebuild]: ah, of course.

**Comment by Hummer12007** [comment]
Let's put it on hold until ambientsound/pms#42 gets merged. Getting some really weird build failures with autoconf, so it's easier to rewrite the whole build system in CMake.

File: media-sound/pms/pms-9999.ebuild [hunk 2]
```diff
@@ -12,45 +12,26 @@ EGIT_REPO_URI="https://github.com/ambientsound/pms.git"
...
-	fi
-}
-
-src_prepare() {
-	eapply "${FILESDIR}/pms-9999-fix-automagic-dep.patch"
-	eapply_user
-
-	eautoreconf
-}
+DOCS=( AUTHORS README.md TODO )
```
Comment (SoapGentoo) [inline media-sound/pms/pms-9999.ebuild]: do you really need these? `einstalldocs` defaults should catch them all

Comment (Hummer12007) [inline media-sound/pms/pms-9999.ebuild]: Fixed

**Pull request closed without merging**